### PR TITLE
add talk block for PHPRUHR talk in november 2016

### PR DESCRIPTION
**Pull request by ElectricMaxxx**

### CODE REVIEW

File: src/AppBundle/Resources/data/block.yml [hunk 1]
```diff
@@ -100,3 +100,13 @@ static:
                 on symfony usergroup frankfurt on the 23th of june 3016. He will suggest the
                 CMF as a way to serve CMS features to a given Symfony application. The
                 talk will be in german only.
+    sfu-june-2016-talk:
```
Comment (dbu) [inline src/AppBundle/Resources/data/block.yml]: please adjust to your event. the key needs to be unique

File: src/AppBundle/Resources/data/block.yml [hunk 2]
```diff
@@ -100,3 +100,13 @@ static:
...
+        class: AppBundle\Document\TalkBlock
+        properties:
+            title: Symfony Usergroup Talk by @ElectricMaxxx in Frankfurt
+            publishEndDate: "2016-11-10 19:00"
+            link: https://2016.php.ruhr/session/extending-a-symfony-application-by-cms-features/
+            body: |
+                Maximilian Berghoff (@ElectricMaxxx) will give a talk about the CMF
+                at the <a href="https://2016.php.ruhr/">PHPRUHR</a> conference on the 10th of november 2016 in dortmund.
```
Comment (dbu) [inline src/AppBundle/Resources/data/block.yml]: afaik city names are uppercase in english, so it would be Dortmund.

**Comment by dbu** [comment]
can you please rebase on master, it looks like you started from an outdated version as there is a conflict.

File: src/AppBundle/Resources/data/block.yml (hunk 1 above)
Comment (ElectricMaxxx) [inline src/AppBundle/Resources/data/block.yml]: Changed, stupid copy-paste

**Comment by ElectricMaxxx** [comment]
it is rebased on master.

**Pull request merged**